src/class-tools/README.md

```diff
@@ -12,7 +15,14 @@ Within a run, a `,` character separates distinct class operations.
 A class operation is an operation name `add`, `remove`, or `toggle`, followed by a CSS class name,
 optionally followed by a colon `:` and a time delay.
 
-There is also the option to use `apply-parent-classes` or `data-apply-parent-classes` which uses the same format as `classes` but is instead designed for Out of band updates to allow you to manipulate CSS classes of an existing element in the DOM without otherwise knowing or altering its state. Any element with this property will apply classes to its parent and also remove this child element afterwards so should ideally be used as part of a `hx-swap-oob="beforeend: #some-element`.
+## Out-of-band class manipulation
+
+There is also the option to use `apply-parent-classes`, or `data-apply-parent-classes`, which take the same format as `classes`
+but is instead designed for out-of-band updates, allowing you to manipulate CSS classes of an existing element in the DOM
+without otherwise knowing or altering its state.
+
+Any element with this property will schedule classes to be applied to its _parent_ element, _removing_ itself afterwards,
+so it should ideally be used as part of an `hx-swap-oob="beforeend: #some-element` to add them to the end of the target element.
```

Comment:
Didn't notice that we've been missing a closing quote all along for this `hx-swap-oob` attribute value example btw, could you add it?

Reply:
Done